tools/PI/DevHome.PI/Strings/en-us/Resources.resw

```diff
@@ -1177,8 +1177,45 @@
     <value>No issues</value>
     <comment>A message indicating that there are no issues with the given item.</comment>
   </data>
+  <data name="ToolTypeButton.Content" xml:space="preserve">
+    <value>Tool Type</value>
+    <comment>Button text allowing the user to select the type of tool</comment>
+  </data>
+  <data name="ToolTypeButton.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Set the type of the tool being added.</value>
+    <comment>Tooltip for the button text allowing the user to select the type of tool</comment>
+  </data>
+  <data name="ToolTypeMenuItemDumpAnalyzer.Text" xml:space="preserve">
+    <value>WER Dump Analyzer</value>
+    <comment>Describes type of tool used to analyze WER dumps</comment>
+  </data>
+  <data name="ToolTypeMenuItemDumpAnalyzer.ToolTipService.ToolTip" xml:space="preserve">
+    <value>Tool that will be used to analyze WER dumps generated on the system</value>
+    <comment>Tooltip that describes what a WER Dump analyzer does</comment>
+  </data>
   <data name="ClipboardMonitorWindowTitle" xml:space="preserve">
     <value>Project Ironsides Clipboard Monitor</value>
     <comment>{Locked="Ironsides"} Window title for the clipboard monitor tool</comment>
   </data>
+  <data name="WERAnalysisUnavailable" xml:space="preserve">
+    <value>Analysis unavailable.</value>
+    <comment>Message shown when there isn't crash analysis available.</comment>
+  </data>
+  <data name="WERBasicInfo" xml:space="preserve">
+    <value>Dump File Size: {0} bytes
+Dump File creation time: {1}</value>

```

Comment:
Any reason for the new line?

Reply:
Yep. When I don't have eventviewer data for a WER dump, I wanted these items on separate lines....

(I don't have an example on my dev box in the office, so imagine this data in the DumpInfo box)

![image](https://github.com/user-attachments/assets/326eaa71-ba7b-4f2d-9a2e-abcf15684357)